src/twisted/tap/ftp.py

```diff
@@ -47,6 +47,17 @@ def opt_password_file(self, filename):
         warnings.warn(msg, category=DeprecationWarning, stacklevel=2)
         self.addChecker(checkers.FilePasswordDB(filename, cache=True))
 
+    def opt_passive_port_range(self, port_range_str):
+        """
+        Two numbers, colon separated, denoting the range of passive ports (both
+        ends inclusive). E.g. "--passive-port-range=30000:31000".
+        """
+        # Let it fail upon invalid user inputs.
+        port_0, port_1 = [int(x) for x in port_range_str.split(":")]
+        assert port_0 <= port_1
```

Comment:
Please use `if` and `raise` rather than `assert`, as `assert` statements are stripped when Python is run with the `-O` flag.